src/systems/camera-tools.js

```diff
@@ -16,6 +16,10 @@ AFRAME.registerSystem("camera-tools", {
     this.myCamera = null;
   },
 
+  avatarUpdated() {
+    this.cameraEls.forEach(el => delete el.components["camera-tool"].playerHead);
```

Comment:
this feels a little magical, maybe push it into a function `onAvatarUpdated` in the camera tool component?